suites/pacific/cephadm/tier_0_cephadm.yaml

```diff
@@ -299,6 +299,21 @@ tests:
             sep: " "
       destroy-cluster: false
       abort-on-fail: true
+  - test:
+      name: Add OSD Service
+      desc: Add OSD service to single devices
+      module: test_osd.py
+      polarion-id:
+      config:
+        command: add
+        service: osd
+        osd_device_count: 3
+        base_cmd_args:
+          verbose: true
+        args:
+          node: node3
+      destroy-cluster: false
+      abort-on-fail: true
```

Comment:
IMO, this test case does not meet the requirements to be part of Tier-0. I believe it be more fit for Tier-1 or Tier-2
@VasishtaShastry @sunilkumarn417 kindly let me know if you disagree with this assessment.

Reply:
> IMO, this test case does not meet the requirements to be part of Tier-0. I believe it be more fit for Tier-1 or Tier-2
> @VasishtaShastry @sunilkumarn417 kindly let me know if you disagree with this assessment.

@psathyan  In tier-0 we are validating `--all-available-devices` which fulfills Tier-0 criteria, so we planned to include this in out Tier-1.